Resources/Prototypes/DeltaV/Entities/Objects/Weapons/Guns/Battery/battery_guns.yml

```
@@ -2,7 +2,7 @@
   name: energy gun
   parent: BaseWeaponBattery
   id: WeaponEnergyGun
-  description: "A basic hybrid energy gun with two settings: disable and kill."
+  description: "A basic hybrid energy gun with two settings: disable and kill.  On the handle is a label that says 'for authorized use only.'"
```

```suggestion
  description: "A basic hybrid energy gun with two settings: disable and kill. On the handle is a label that says 'for authorized use only.'"
```
Double space after period here, most descriptions use single space.